wiki/en/Running-a-Server.md

```diff
@@ -125,185 +144,174 @@ To register with one of the Directories built into the Jamulus Client, replace `
 
-For starting a new recording:
+##### `-d or --discononquit` 
+Normally, when a Server is stopped or restarted, any Clients that have not used their “Disconnect” buttons will re-establish connection when the Server comes back up again. Using this option forces Clients to manually re-establish their connections to the Server.
 
-~~~
- [Unit]
- Description=Start a new recording on Jamulus Server
- Requisite=Jamulus-Server
+##### `-F or --fastupdate` 
+Reduces latency if Clients connect with “Enable Small Network Buffers” option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.
 
- [Service]
- Type=oneshot
- ExecStart=/bin/systemctl kill -s SIGUSR1 Jamulus-Server
-~~~
+##### `-l or --log` 
+Enable logging, set path and file name
 
-_Note: The Jamulus service name in the `ExecStart` line needs to be the same as the `.service` file name you created when setting systemd to control your Jamulus Server. So in this example it would be `Jamulus-Server.service`_
+##### `-m or --htmlstatus` 
+Enable HTML status file, set path and file name
 
-Run `sudo systemctl daemon-reload` to register them for first use.
+##### `-P or --delaypan` 
+Start with delay panning enabled.This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal “volume” panning.
 
-Now you can run these with the `systemctl` command, for example:
+##### `-s` or `--server`
+Start Jamulus in Server mode
 
-`sudo systemctl start jamulusTogglerec` (assuming you named your unit file `jamulusTogglerec.service`)
+##### `--serverbindip` 
+Specify the IP address the Jamulus process will bind to
 
-You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs.
+##### `-T or --multithreading` 
+Use multithreading to make better use of multi-core CPUs to support more Clients
 
+##### `-u or --numchannels` 
+Maximum number of channels (Clients)
 
-## Servers on the desktop
+##### `-z or --startminimized` 
+Start the graphicial user interface minimized
 
-Jamulus can be run in Server mode from the desktop. This gives you a graphical user interface to control most of the settings.
+---
 
-* **Windows users** - Use the "Jamulus Server" icon in the Windows Start menu.
-* **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
-* **Linux users** - Launch the "Jamulus Server" shortcut. Or you can open a terminal window (`CTRL+ALT+t` on Debian and related distros), type `jamulus -s` and hit return.
+## Other options
 
+{% include_relative Include-Shared-Commands.md %}
 
-## Server Setup
+---
 
-<figure><img src="{% include img/en-screenshots/server-window-setup.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of the Jamulus Server setup window"></figure>
+## Recording
```

Comment:
Maybe this could all go into a separate file?  I'm not sure how commonly used it is.  Just to cut down the content.  (Like there's a Directory guide, I was thinking.)

Reply:
Thought about that, but my thoughts a above re. the options associated with it: https://github.com/jamulussoftware/jamuluswebsite/pull/877#discussion_r1035328241